### ISSUE TITLE
Proper way to launch a page directly with brave from the cli on linux? 

### ISSUE THREAD

**Issue by bennyprofane** [brave/browser-laptop#12127]
### Description

I would like to know the proper way to launch the brave browser from the command line on linux and have it open to a specific page. With chromium and google chrome you can do the following:

`$ chromium 'gentoo.org'`
`$ google-chrome-stable 'gentoo.org'` 

and it will launch the browser and open it to the link you specify, in my example gentoo.org. When I attempt this with brave:

`$ brave-bin 'gentoo.org'` 

It just launches brave and doesn't open gentoo.org. 

One of the main reasons I would like this functionality is that I use the `mutt` mail client and you can set a browser as your default in your `~/.mailcap` file and it allows you to open email html links in that browser. The mailcap code I use for chromium looks like this:

`text/html; chromium '%s' &; test=test -n "$DISPLAY"; needsterminal;`

I would like to use brave instead of chromium but I don't know the proper way to get this result with brave, or if it's even possible to currently do this with brave.  

### Steps to Reproduce

  1. Launch brave from a terminal on linux (i'm using gentoo) with website as an argument: `$ brave-bin 'gentoo.org'`

**Actual result:**

It will launch brave but will not open gentoo.org.

**Expected result:**

Expected result is that it will both launch brave browser and open to the page specified as an argument. The expected result can be seen when launching chromium or google-chrome

`$ google-chrome-stable 'gentoo.org'`
`$ chromium 'gentoo.org'` 

both of the above commands on my system launch the browser and open to gentoo.org. 

**Reproduces how often:**

this happens 100% of the time. 

### Brave Version
**about:brave info:**

Name	Version
Brave	0.19.95
rev	cc0ebad
Muon	4.5.16
libchromiumcontent	62.0.3202.94
V8	6.2.414.42
Node.js	7.9.0
Update Channel	Release
OS Platform	Linux
OS Release	4.9.34-gentoo
OS Architecture	x64
**Reproducible on current live release:**

**Comment by bsclifton** [comment]
@bennyprofane what window manager are you using? Installing Brave will update the `.desktop` files which have the parameters passed to Brave... but I know this has issues on XFCE and KDE (and likely many others)

You may check out the conversation here for some hints- let us know!
https://github.com/brave/browser-laptop/issues/9193

**Comment by bennyprofane** [comment]
I'm just using openbox and I launch all my applications from the terminal. So when I normally launch brave I do it as `$ brave-bin`. I don't have any icons that I click on or anything.

I looked through the conversation you linked and I noticed I didn't even have a `brave-bin.desktop` file so I added one I tried trying the different `Exec=` variations but I still haven't gotten it to work. I also checked and I don't have a `.desktop` file for chromium but I do have one for google chrome. What exactly is that `Exec` parameter supposed to do? 

Running the different `Exec=` command permutations directly also doesn't create the results I want:

`$ sh -c "brave-bin gentoo.org"`
`$ brave-bin  -- gentoo.org`
`$ sh -c "/usr/brave-bin -- gentoo.org"`
`$ /usr/bin/brave-bin gentoo.org`
`$ /usr/bin/brave-bin -- gentoo.org`

The above all launch brave but don't open to gentoo.org

I may be wrong but why would the `.desktop` file have any effect over my direct cli commands? I never launch apps from any menus or icons or anything. I launch everything from the terminal. So I don't see how the issue is `.desktop` file related other than if this `.desktop` file is being read when I run the `/usr/bin/brave-bin` executable. Shouldn't the ability to pass a parameter work directly without the need of this `Exec=` line in the `.desktop` file? 

What's also weird is that the `--help` flag doesn't work for me but I have been able to successfully use the `--proxy-server=` command like `$ brave-bin --proxy-server="socks5://localhost:8000"`. This command I found by trying the command that chromium and google-chrome uses. So that would be an example of a direct command line flag and parameter that is working for me that didn't require a `.desktop` file.
